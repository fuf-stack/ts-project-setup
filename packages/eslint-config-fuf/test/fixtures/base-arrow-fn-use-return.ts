/* eslint-disable */

// use braces and return in arrow fn
const testFn = () => 'test';

testFn();
