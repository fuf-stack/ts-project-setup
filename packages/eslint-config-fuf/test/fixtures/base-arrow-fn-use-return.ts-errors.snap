[
  {
    "message": "Expected block statement surrounding arrow body.",
    "ruleId": "arrow-body-style",
  },
]